### ISSUE TITLE
[Counties] Add drag and drop option to counties list

### ISSUE THREAD

**Issue by aniri** [code4romania/monitorizare-vot-ong#221]
Depends on https://github.com/code4romania/monitorizare-vot-ong/issues/215

Add the possibility of reordering the counties by drag and dropping the items from the counties table.

UI available here: https://www.figma.com/file/61NHR7fwmV1Dsv4Pb6F8T7/code4ro_mv_user-management_2018.04.21-(Copy)?node-id=754%3A0

Linked to UI issue for styleguide #209 . It would be best if the styleguide issues were implemented first so the elements could be reused here.

**Comment by Botosio** [comment]
Assign to me and I can take a look

**Comment by Botosio** [comment]
Can you add hacktoberfest to the label also?

**Comment by Botosio** [comment]
@aniri do you want every time someone drags and drops to send an update to the backend? 
Do we need a save button? 

I'm guessing every time they drag and drop we will save it.

**Comment by Botosio** [comment]
@aniri what happens if you filter the list and they change the order?

Right now I disabled the drag and drop if the filter is on.

If you filter, the index doesn't match the original order.  However, the move to first when the filter is on should be able to work.

**Comment by Botosio** [comment]
@aniri do you have this icon?
![image](https://user-images.githubusercontent.com/44526601/138769314-14b0963c-8b5c-4a68-9f2c-f951037df462.png)

**Comment by aniri** [comment]
hey @Botosio 

> I'm guessing every time they drag and drop we will save it.
Yep, sounds good!

Also good idea with disabling the drag and drop when the filter is on!

Here's the icon :D 
![double up](https://user-images.githubusercontent.com/1358055/138825078-704c53e1-7793-404a-ae46-2a1b31b3691e.png)

Thank you!

**Comment by Botosio** [comment]
Should there be a post without the ID for new Counties?

![image](https://user-images.githubusercontent.com/44526601/140051251-03df0037-50ff-4404-82f8-07b6af9fd6a1.png)

**Comment by aniri** [comment]
yes, it should be a post without id for adding a county and a put with id for editing one.

I'll add a backend issue to adjust these ;) thanks!

**Comment by Botosio** [comment]
@aniri sounds good. I made the global change in this next pr to change the word "Country" to "County".... I might have been the culprit in that.

**Comment by Botosio** [comment]
@aniri we need a delete also, for county

**Comment by aniri** [comment]
@Botosio thanks for the suggestion! I added a backend issue for all the mentioned changes :D

**Comment by aniri** [comment]
depends on https://github.com/code4romania/monitorizare-vot/issues/388
